### PR TITLE
Improve test factories and dummy data

### PR DESCRIPTION
**Pull request by pitpalme**

### CODE REVIEW

File: tests/factories.py [hunk 1]
```diff
@@ -12,109 +13,120 @@
...
     class Meta:
         model = User
 
     username = FuzzyText(length=10, chars=string.ascii_letters, prefix='')
     first_name = FuzzyText(length=10, chars=string.ascii_letters, prefix='')
     last_name = FuzzyText(length=10, chars=string.ascii_letters, prefix='')
-    password = factory.PostGenerationMethodCall('set_password', 'defaultpassword')
-    email = factory.LazyAttribute(lambda o: '%s@example.com' % o.last_name)
+    password = PostGenerationMethodCall('set_password', 'defaultpassword')
+    email = LazyAttribute(lambda o: '%s@example.com' % o.last_name)
```
Comment (christophmeissner) [inline tests/factories.py:116]: Replace with an f-string?

```suggestion
    email = LazyAttribute(lambda o: f"{o.last_name}@example.com")
```

File: tests/factories.py [hunk 2]
```diff
@@ -12,109 +13,120 @@
...
         model = scheduler_models.Shift
 
-    task = factory.SubFactory(TaskFactory)
-    facility = factory.SubFactory(FacilityFactory)
+    task = SubFactory(TaskFactory)
+    facility = SubFactory(FacilityFactory)
+    workplace = SubFactory(WorkplaceFactory)
 
     starting_time = datetime(2016, 2, 13, 19, 0)
     ending_time = datetime(2016, 2, 13, 20, 0)
```
Comment (christophmeissner) [inline tests/factories.py:104]: Change to datetime.now or sth. in that ballpark?

File: scheduler/management/commands/create_dummy_data.py [hunk 3]
```diff
@@ -61,47 +62,51 @@ def handle(self, *args, **options):
...
-
-                UserAccount.objects.all().delete()
-
-                # delete geographic information
-                Place.objects.all().delete()
-                Area.objects.all().delete()
-                Region.objects.all().delete()
-                Country.objects.all().delete()
-
+                for clazz in [RegistrationProfile, ShiftHelper, Shift, UserAccount, Task, Workplace, Facility,
```
[the remark this answers is not in the document]
Comment (pitpalme) [reply scheduler/management/commands/create_dummy_data.py:65]: Caught me - not yet a djangonaut, but can't deny my heritage. :D

[the remark this answers is not in the document]
Comment (pitpalme) [reply scheduler/management/commands/create_dummy_data.py:65]: It's gonna be changed and out of curiosity: what's the difference in handling / behaviour?
Is it only about "not being able to alter this anonymous enlisting of values"?

File: tests/factories.py (hunk 1 above)
Comment (pitpalme) [reply tests/factories.py:116]: *meeehhhh* didn't touch too much code, but: valid point.

File: scheduler/management/commands/create_dummy_data.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (pitpalme) [reply scheduler/management/commands/create_dummy_data.py]: Changed

[the remark this answers is not in the document]
Comment (christophmeissner) [reply scheduler/management/commands/create_dummy_data.py]: In fact, parenthesis for enumeration of model classes can be omitted entirely and python will handle it as a tuple. Parenthesis only become required, when the enumeration is formatted to span multiple lines. 

However, my rule of thumb is: when it's not required to be mutable, it should not be mutable. Most of the times, mutable objects are at least slightly more expensive. This article breaks the implications of (im-)mutability pretty good in a few paragraphs: https://medium.datadriveninvestor.com/mutable-and-immutable-python-2093deeac8d9

File: tests/factories.py [hunk 4]
```diff
@@ -86,8 +86,8 @@ class WorkplaceFactory(DjangoModelFactory):
     class Meta:
         model = organization_models.Workplace
 
-    name = Sequence(lambda n: 'Workplace ' + str(n))
-    description = Sequence(lambda n: f'workplace {n}')
+    name = Sequence(lambda n: "fWorkplace {n}")
```
Comment (christophmeissner) [inline tests/factories.py:89]: Typo in f-string
```suggestion
    name = Sequence(lambda n: f"Workplace {n}")
```

**Review by christophmeissner — CHANGES_REQUESTED** [review]
Just one more thing ;-)

**Comment by pitpalme** [comment]
> Just one more thing ;-)

*duuuhh* another kind of "one-off" 😆

**Review by christophmeissner — APPROVED** [review]

**Pull request merged**